### PR TITLE
Support live contribution and audio description profiles

### LINKED ISSUE #34: Update to address tt-reqs agreed new requirements and structure
A number of new requirements for TTWG in 2019 were agreed, as given the milestone "[Finalised 2019 Requirements]( https://github.com/w3c/tt-reqs/milestone/2)"; 2 were labelled as [requires-charter-revision](https://github.com/w3c/tt-reqs/issues?q=is%3Aopen+is%3Aissue+label%3Arequires-charter-revision). Address those two in the draft charter:

* [x] Audio Description profile of TTML2 w3c/tt-reqs#4
* [x] Support live contribution of TTML w3c/tt-reqs#3

### PR DESCRIPTION
**Pull request by nigelmegitt**
Closes #34 noting that if a member submission is incoming e.g. for live streaming of TTML, further work will be needed to reference it in the draft Charter.

### CODE REVIEW

**Comment by andreastai** [comment]
@nigelmegitt I can see the part on AD but I am missing the part on live contribution in this PR. Can you point me to it?

File: index.html [hunk 1]
```diff
@@ -146,8 +146,9 @@ <h2>Scope</h2>
                 however, it may emit warnings if it encounters deprecated
                 features.</li>
               <li>Facilitate mapping to HTML5/CSS3.</li>
-              <li>Should address the <a href="https://www.w3.org/TR/media-accessibility-reqs/">Media
+              <li>Address the <a href="https://www.w3.org/TR/media-accessibility-reqs/">Media
                   Accessibility User Requirements</a>.</li>
+              <li>Facilitate contribution of real time streaming.</li>
```
Comment (nigelmegitt) [inline index.html]: @tairt concern about potential misunderstanding or confusion with e.g. DASH. Request rewording to narrow scope to contribution between authoring and distribution encoders for example. See summary from EBU Tech3370.

File: index.html [hunk 2]
```diff
@@ -291,6 +314,30 @@ <h3> Normative Specifications </h3>
...
                 2018<br>
                 Produced under <a href="https://www.w3.org/2016/05/timed-text-charter.html">2016 Working Group Charter</a></dd>
+                <br/>
+              <dt id="adpt" class="spec">TTML Profile for Audio Description</dt>
+              <dd><p>This specification defines a profile of 
+                [<cite><a class="bibref" href="https://www.w3.org/TR/ttml-imsc1.1/#bib-TTML2">TTML2</a></cite>]
+                intended to support audio description script exchange
+                throughout the workflow from production (scripting)
+                through to distribution
+                (mixing either at the broadcaster
```
Comment (nigelmegitt) [inline index.html]: s/broadcaster/something broader.

File: index.html [hunk 3]
```diff
@@ -291,6 +314,30 @@ <h3> Normative Specifications </h3>
                 Associated <a href="https://lists.w3.org/Archives/Member/member-cfe/2018May/0003.html">Call for Exclusion</a> on 10 May 2018, ending on 9 July
                 2018<br>
                 Produced under <a href="https://www.w3.org/2016/05/timed-text-charter.html">2016 Working Group Charter</a></dd>
+                <br/>
+              <dt id="adpt" class="spec">TTML Profile for Audio Description</dt>
+              <dd><p>This specification defines a profile of 
+                [<cite><a class="bibref" href="https://www.w3.org/TR/ttml-imsc1.1/#bib-TTML2">TTML2</a></cite>]
```
Comment (nigelmegitt) [inline index.html]: Citation is wrong, needs to point to TTML generic URL `https://www.w3.org/TR/ttml/`, and omit TTML version number.

File: index.html (hunk 1 above)
Comment (nigelmegitt) [reply index.html]: @tairt I think I've addressed this - please re-review.

**Pull request merged**